checks.d/openstack.py

```diff
@@ -512,3 +650,57 @@ def check(self, instance):
         except IncompleteConfig:
             self.warning("Configuration Incomplete! Check your openstack.yaml file")
             return
+
+
+    #### Local Info accessors
+    def get_local_hypervisor(self):
+        # Look up hypervisors available filtered by my hostname
+        host = self.get_my_hostname()
+        hyp = self.get_all_hypervisor_ids(filter_by_host=host)
+        return hyp[0]
+
+    def get_local_project(self):
```

Comment:
I'm not sure I understand this method. Projects by definition are global and not specific to a hypervisor.